articles/libraries/overview.md

```diff
@@ -37,7 +37,7 @@ The Lock widget is a simple way to integrate Auth0 into existing projects and pr
 These client-side SDKs for Auth0 allow you to trigger the authentication process, parse JWTs, and authenticate requests to your APIs. There is no UI or widget provided, so these SDKs are for use when you need to implement a custom UI, or already have one that you'd like to integrate Auth0 into.
 
 <%= include('../_includes/_topic-links', { links: [
-  'libraries/auth0js',
+  'libraries/auth0js/v8',
```

Comment:
Without this change the build keeps failing with this error:

```
[2017-03-22T11:33:08.914Z] ERROR: auth0-docs/8442 on Marias-MBP.lan: Uncaught Exception (log_type=uncaughtException, uptime=8.175, err.constructor=Error)
    Error: While rendering document articles/libraries/overview: While rendering include /Users/mariap/auth0-docs-master/docs/articles/_includes/_topic-links.md: No document with the path articles/libraries/auth0js exists in the cache
        at Cache.get (/Users/mariap/auth0-docs-master/lib/pipeline/Cache.js:96:21)
        at Object.get (/Users/mariap/auth0-docs-master/lib/pipeline/Compiler.js:116:27)
        at eval (lodash.templateSources[153]:65:23)
        at arrayEach (/Users/mariap/auth0-docs-master/node_modules/lodash/lodash.js:521:11)
        at Function.forEach (/Users/mariap/auth0-docs-master/node_modules/lodash/lodash.js:9252:14)
        at eval (lodash.templateSources[153]:64:4)
        at Document.doc.render (/Users/mariap/auth0-docs-master/lib/pipeline/Compiler.js:79:31)
        at Document.getContent (/Users/mariap/auth0-docs-master/lib/pipeline/models/Document.js:80:29)
        at /Users/mariap/auth0-docs-master/lib/pipeline/Cache.js:84:11
        at Array.forEach (native)
```

This looks like a bug, so please open an issue at [auth0-docs repo](https://github.com/auth0/auth0-docs/issues). Till this is fixed you can leave this temporary fix in place.